### PR TITLE
rely on transaction rollback for cleanup

### PR DESCRIPTION
**Pull request by nickpell**
testing if this works because of [transactions](https://docs.djangoproject.com/en/1.10/topics/testing/tools/#testcase)

@emord

### CODE REVIEW

File: corehq/apps/hqadmin/tests/test_prbac.py [hunk 1]
```diff
@@ -28,23 +28,10 @@ class TestCchqPrbacBootstrap(TestCase):
 
     @classmethod
     def setUpClass(cls):
+        super(TestCchqPrbacBootstrap, cls).setUpClass()
         Grant.objects.all().delete()
```
Comment (emord) [inline corehq/apps/hqadmin/tests/test_prbac.py]: Would this work if you got rid of these teardowns in the setup?

**Review by emord — COMMENTED** [review]
Did you test this locally by running this test and inspecting the database afterwards? I think that just because this test passed doesn't mean that it doesn't leave around some artifacts that could make a future test fail

File: corehq/apps/hqadmin/tests/test_prbac.py (hunk 1 above)
Comment (nickpell) [inline corehq/apps/hqadmin/tests/test_prbac.py]: That should break tests.  When I make these edits tests still pass:
```
diff --git a/corehq/apps/hqadmin/tests/test_prbac.py b/corehq/apps/hqadmin/tests/test_prbac.py
index 61a4d95..296b8f7 100644
--- a/corehq/apps/hqadmin/tests/test_prbac.py
+++ b/corehq/apps/hqadmin/tests/test_prbac.py
@@ -32,6 +32,14 @@ class TestCchqPrbacBootstrap(TestCase):
         Grant.objects.all().delete()
         Role.objects.all().delete()
 
+    @classmethod
+    def tearDownClass(cls):
+        assert Role.objects.count() == 0
+        assert Grant.objects.count() == 0
+        super(TestCchqPrbacBootstrap, cls).tearDownClass()
+        assert Role.objects.count() != 0
+        assert Grant.objects.count() != 0
+
     def test_dry_run(self):
         """
         When --dry-run is passed, no models should be created
```

which looks to be, before the superclass ```tearDownClass``` is called, 
```
Grant.objects.all().delete()
Role.objects.all().delete()
```
still has an effect, and after ```tearDownClass``` is executed, the transaction opened in ```setUpClass``` is rolled back, so the rows reappear in the DB.  Which is consistent with my interpretation of transactions in setup(class)/teardown(class)

Comment (emord) [inline corehq/apps/hqadmin/tests/test_prbac.py]: Ah. ok so those are needed for some random unclean shutdowns elsewhere. thanks for looking into that

**Pull request merged**